### PR TITLE
Adding pyModelChecking

### PR DESCRIPTION
**Pull request by xtrojak**
This PR adds a recipe for [pyModelChecking](https://github.com/albertocasagrande/pyModelChecking).

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by xtrojak** [comment]
@conda-forge-admin, please restart ci

**Comment by xtrojak** [comment]
@conda-forge-admin, please ping team

**Review by BastianZim — CHANGES_REQUESTED** [review]

File: recipes/pymodelchecking/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/albertocasagrande/pyModelChecking
+  summary: A simple Python model checking package
+  license: MIT
+  license_file: LICENSE
```
[the remark this answers is not in the document]
Comment (BastianZim) [reply recipes/pymodelchecking/meta.yaml:37]: I have created https://github.com/albertocasagrande/pyModelChecking/pull/6 to have it included, until then you can just manually copy the file here.

File: recipes/pymodelchecking/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    - pyModelChecking
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/albertocasagrande/pyModelChecking
+  summary: A simple Python model checking package
+  license: MIT
```
[the remark this answers is not in the document]
Comment (xtrojak) [reply recipes/pymodelchecking/meta.yaml]: Yes you are right, I didn't notice `grayskull` generated the recipe with `MIT` licence.

File: recipes/pymodelchecking/meta.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (xtrojak) [reply recipes/pymodelchecking/meta.yaml:37]: Thank you for noticing this and creating the PR in the upstream tool. I added the license file in [9c8cc24](https://github.com/conda-forge/staged-recipes/pull/17978/commits/9c8cc24538f149f2eba4c029a5f9866573dd6c00).

**Pull request merged**